content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: blog
+title: 'Kubernetes v1.30: TBA'
+date: 2024-04-17
+slug: kubernetes-v1-30-release
+---
+
+**Authors:** [Kubernetes v1.30 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release_team.md)
+
+**Editors:** Amit Dsouza, Frederick Kautz, Kristin Martin, Abigail McCarthy, Natali Vlatko
+
+Announcing the release of Kubernetes v1.30: Theme TBA, the first release of 2024!
+
+Similar to previous releases, the release of Kubernetes v1.30 introduces new stable, beta, and alpha
+features. The consistent delivery of top-notch releases underscores the strength of our development
+cycle and the vibrant support from our community.
+
+This release consists of XX enhancements. Of those enhancements, XX have graduated to Stable, XX are
+entering Beta, and XX have graduated to Alpha.
+
+
+## Release theme and logo
+<Logo image size is recommended to be no more than 2160px> 
+[Kat requested to hold review for this section until after the theme is revealed, and other 
+sections are reviewed.]
+
+
+## Improvements that graduated to stable in Kubernetes v1.30 {#graduations-to-stable}
+
+_This is a selection of some of the improvements that are now stable following the v1.30 release._
+
+### Robust VolumeManager reconstruction after kubelet restart ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+Blurb for 3756 goes here.
+
```

Comment:
Suggestion:

This is a volume manager refactoring that allows the kubelet to populate additional information about how existing volumes are mounted during the kubelet startup. In general, this makes volume cleanup after kubelet restart or machine reboot more robust.

This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is GA, the feature gate is locked and cannot be disabled.

Reply:
thanks @jsafrane! i've added this into the PR.